src/RNCloudStorage.ts

```diff
@@ -92,6 +92,16 @@ const RNCloudStorage = {
     return nativeInstance.readFile(verifyLeadingSlash(path), scope ?? defaultScope);
   },
 
+  /**
+   * Trigger a download of the file at the given path.
+   * @param path The file to trigger the download for.
+   * @param scope The directory scope the path is in. Defaults to the set default scope.
+   * @returns A promise that resolves to true if the download was triggered, false otherwise.
```

Comment:
It would be good to specify in which cases this can return `false`. Are there any cases where this could return `false` when using the library? I'd suppose the native function (`isUbiquitousItem(at:)`) only returns false when a local file path is given, which shouldn't be possible in the scope of this library?